### ISSUE TITLE
PancakeView + XamarinForms issue

### ISSUE THREAD

**Issue by ysmoradi** [unoplatform/uno#2322]
## Current behavior

![image](https://user-images.githubusercontent.com/5352980/70852979-41370b00-1ebd-11ea-8771-d994d8c08e8c.png)

The left side is uwp and right side is wasm, these are not identical )-:

## Expected behavior

These should be identical

## How to reproduce it (as minimally and precisely as possible)

[RpnCalculator.zip](https://github.com/unoplatform/uno/files/3964186/RpnCalculator.zip)

I've downloaded RpnCalculator, then I updated all nuget packages and added [PanCakeView](https://github.com/sthewissen/Xamarin.Forms.PancakeView)

Note that you've to add
```xml
<AssetTargetFallback>uap10.0.16299</AssetTargetFallback>
```
to wasm csproj

## Environment

Nuget Package:

Package Version(s):

```xml
    <PackageReference Include="Uno.UI" Version="2.1.0-dev.272" />
    <PackageReference Include="Uno.Wasm.Bootstrap" Version="1.0.8" />
    <PackageReference Include="Uno.Xamarin.Forms.Platform" Version="4.3.0-uno.141" />
    <PackageReference Include="Xamarin.Forms.PancakeView" Version="1.3.6" />
    <DotNetCliToolReference Include="Uno.Wasm.Bootstrap.Cli" Version="1.0.10" />
```

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [X] WebAssembly renderers for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [X] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?

It works if you copy [renderer code](https://github.com/sthewissen/Xamarin.Forms.PancakeView/blob/master/src/Xamarin.Forms.PancakeView.Multi/Platforms/UWP/PancakeViewRenderer.cs) into the project instead of installing the package into the project. Note that I've add Xamarin.Forms.PancakeView to LinkerDescription.xml, but it wasn't enough )-:

**Comment by ysmoradi** [comment]
It was my fault, it's working )-: So sorry

**Comment by ysmoradi** [comment]
For anyone who is interested, take a look at https://github.com/ysmoradi/UnoXamWasm

**Comment by jeromelaban** [comment]
@ysmoradi very nice!! What did not work in the first place, is it something we need to adjust ?

**Comment by ysmoradi** [comment]
I invested a time to understand how Uno works and I should say, it's fantastic.
There are some issues with installing 3rd party nuget packages although. Pancake itself works if I copy it's renderer code to the project, but I was unable to achieve the same by installing nuget package. I tried to install some other nuget packages such as syncfusion components. The fact that wasm project is a .net standard 2.0 project, prevents installing and using those packages. I'm almost sure they work if I copy their renderer codes, but it's not possible in the real world. I'm thinking about how to solve it and I know dot net sdk has its own limitations. For now, I closed this issue, because there is nothing wrong with Uno and Pancake, but there are more important issues which needs to be discussed about toolings.
I'll open 2 new issues tomorrow (another one is related to gradiant feature of pancake)
I think, at the end, it should be possible to pirt a lot of existing xamarin forms project to web thanks to uno ✌️
